📖 Update root README file (#541)
metal3-io/cluster-api-provider-metal3: README.md

|---------------|---------------------|---------------|
| v1alpha4      | v1alpha3            | v0.4.X        |
| v1alpha5      | v1alpha4            | v0.5.X        |
| v1beta1       | v1beta1             | v1.1.X        |

## Deploying the metal3 provider

The recommended method is using
[Clusterctl](https://main.cluster-api.sigs.k8s.io/clusterctl/overview.html).

Starting from `v0.5.0` release of Cluster API Provider Metal3, Baremetal Operator is decoupled

> from Cluster API Provider Metal3 deployments when deployed via `clusterctl`. For that reason,

Reviewer: just small nit. Change is optional.
```suggestion
from Cluster API Provider Metal3 deployments when deployed via `clusterctl`. For this reason,
```

Baremetal Operator will not be installed when initializing the Metal3 provider with clusterctl,
and its CRDs and controller need to be manually installed. Example flow of installing Metal3
provider:

1. Install Cluster API core, bootstrap and control-plane providers. This will also install

>   a cert-manager if it is not already installed. To have more verbose logs you can use the -v flag

Reviewer: Better if it is 
```suggestion
  cert-manager if it is not already installed. To have more verbose logs you can use the -v flag
```

Author: Thanks, fixed.

1. Install Ironic. There are a couple of ways to do it.
    - Run within a Kubernetes cluster as a pod, refer to [deploy.sh](https://github.com/metal3-io/baremetal-operator/blob/main/tools/deploy.sh)
      script.

>     - Outside of a Kubernetes cluster as a container. Please refer to [run_local_ironic.sh](https://github.com/metal3-io/baremetal-operator/blob/main/tools/run_local_ironic.sh) script.

Reviewer 2: nit
```suggestion
    - Outside of a Kubernetes cluster as a container. Please refer to the [run_local_ironic.sh](https://github.com/metal3-io/baremetal-operator/blob/main/tools/run_local_ironic.sh) script.
```

Please refer to the [getting-started](docs/getting-started.md) for more info.

## Pivoting ⚠️

Starting from `v0.5.0` release of Cluster API Provider Metal3, Baremetal Operator is decoupled
from Cluster API Provider Metal3 deployments when deployed via `clusterctl`. For that reason,
when performing `clusterctl move`, custom objects outside of the Cluster API chain or not part

> of CAPM3 will not be pivoted to a target cluster. An example of those objects is BareMetalHost, it's

Reviewer 2: I think it should be "its" here, but maybe we could change it a bit to clarify this is not just about BMO resources?
```suggestion
of CAPM3 will not be pivoted to a target cluster. An example of those objects is BareMetalHost, or user created ConfigMaps and Secrets.
```

Author: Thanks, fixed.
